guides/source/active_storage_overview.md

```diff
@@ -332,7 +332,7 @@ To enable variants, add `mini_magick` to your `Gemfile`:
 gem 'mini_magick'
 ```
 
-When the browser hits the variant URL, ActiveStorage will lazy transform the
+When the browser hits the variant URL, Active Storage will lazy transform the
```

Comment:
Still have ActiveStorage in the guide.
https://github.com/rails/rails/pull/31484/files#diff-e0a7f9274451f039571959ee77f804baL511

Reply:
You have a point. Thanks. I've fixed it.